include/bluetooth/gatt.h

```diff
@@ -1145,6 +1145,10 @@ struct bt_gatt_discover_params {
 	uint16_t end_handle;
 	/** Discover type */
 	uint8_t type;
+#if defined(CONFIG_BT_GATT_AUTO_DISCOVER_CCC)
+	/** The sub_params, if any, used for automatic discover */
```

Comment:
This parameter shouldn't be set by the application, right?
It should be documented.

`/** Only for stack-internal use, used for automatic discovery. */`

Reply:
Correct. Updated the comment.